src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -39,17 +39,33 @@ public function __construct(string $manifestPath)
      * With a manifest, we don't really know or care about what
      * the version is. Instead, this returns the path to the
      * versioned file.
+     *
+     * @param string $path
+     *
+     * @return string
```

Comment:
Can be removed as well, we don't add phpdocs for strings.

Reply:
may I ask why? string parameters are as valid as other parameters, arent't they? apart from that there's already 18 cases of `@param string` in the Asset bundle...